zmsadmin/js/page/availabilityDay/form/content.js

```
@@ -9,50 +9,59 @@ import ErrorBar from '../errorBar'
                                 />
                                 <Description attributes={{ "id": "help_AvDayDescription" }}>Optionale Angabe zur Kennzeichnung des Termins. {data.id > 0 ? "Die ID der Öffnungszeit ist " + data.id : " Die Öffnungszeit hat noch keine ID"}.
                                 </Description>
                             </Controls>
                         </FormGroup>
                         <FormGroup>
-                            <Label attributes={{"htmlFor": "AvDayType"}}>Typ</Label>
+                            <Label attributes={{ "htmlFor": "AvDayType" }}>Typ</Label>
                             <Controls>
                                 <Inputs.Select name="type"
-                                    attributes={{ disabled: data.id ? 'disabled' : null, "id": "AvDayType" }}
+                                    attributes={{ disabled: data.id ? 'disabled' : null, "id": "AvDayType", "disabled": inputDisabled }}
```

Biome (1.9.4)

[error] This property value named disabled is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named disabled


(lint/suspicious/noDuplicateObjectKeys)